rules/library.bzl

```diff
@@ -563,7 +563,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
 
     objc_copts.append("-I.")
 
-    objc_copts.extend(("-index-store-path", "$(GENDIR)/rules_ios_apple_library_objc.indexstore"))
+    objc_copts.extend(("-index-store-path", "$(GENDIR)/rules_ios_objc_library_{}.indexstore".format(objc_libname)))
```

Comment:
this will resolve to the same name for targets with the same name in different packages -- is that going to be an issue?

Reply:
oh yeah :( Well, that wouldn't create any problems (after all, before we were grouping all the units in the same directory), but it is a bit confusing to someone debugging why certain unit files don't show up where we want them to.

Let me see if I can add package in here.